.github/workflows/ci.yml

```diff
@@ -2,6 +2,8 @@ name: CI
 on:
   push:
   workflow_dispatch:
+  schedule:
+  - cron: '0 1 * * *'
```

Comment:
Let's run the cookiecutter's own `ci.yml` nightly as we do for other projects, so we get notified if it breaks.